dist/ami/files/scylla_install_ami

```diff
@@ -103,3 +103,5 @@ if __name__ == '__main__':
     with open('/etc/cloud/cloud.cfg', 'w') as f:
         f.write(cfg2)
     shutil.move('{}/99_user_alias.cfg'.format(homedir), '/etc/cloud/cloud.cfg.d')
+    os.makedirs('/home/scyllaadm')
```

Comment:
Is this created with the correct ownership?

Reply:
It was dirty hack to make .bash_profile provided by current scylla-ami since the pacckage install .bash_profile for centos user, I reverted the changes and sent pull-req for scylla-ami:
https://github.com/scylladb/scylla-ami/pull/56